.github/workflows/import-figma.yml

```diff
@@ -27,22 +27,12 @@ jobs:
       # we need to separately import the themes because they are all needed in different formats and selectors
       - name: 🎨 Import onyx variables
         run: |
-          pnpm run @sit-onyx/figma-utils import-variables -k "${{ vars.FIGMA_FILE_KEY }}" -t "${{ secrets.FIGMA_TOKEN }}" -d "../sit-onyx/src/styles/variables/themes" -m onyx -f CSS -s ":where(:root), .onyx-theme-default"
+          pnpm run @sit-onyx/figma-utils import-variables -k "${{ vars.FIGMA_FILE_KEY }}" -t "${{ secrets.FIGMA_TOKEN }}" -d "../sit-onyx/src/styles/variables/themes" -m Value onyx-light onyx-dark -f CSS -s ":where(:root), .onyx-theme-default"
```

Comment:
Should we talk with Jannick about renaming "Value"  to lowercase so its the same as the other themes?